### PR TITLE
Upgrade workflow for 1.26 prep

### PR DESCRIPTION
**Pull request by martin-yang1234**
Ticket: [SRE-3104](https://thescore.atlassian.net/browse/SRE-3104)

Update the version of Pluto and action-setup-kube-tools to the latest version

There's no breaking change b/w the old versions and the latest versions

[SRE-3104]: https://thescore.atlassian.net/browse/SRE-3104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/pluto-workflow-kustomize.yaml [hunk 1]
```diff
@@ -17,11 +17,11 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v3
       - name: Download Pluto
-        uses: FairwindsOps/pluto/github-action@v5.11.2
+        uses: FairwindsOps/pluto/github-action@v5.19.0
```
[the remark this answers is not in the document]
Comment (martin-yang1234) [reply .github/workflows/pluto-workflow-kustomize.yaml:20]: #cycode_ignore_insecure_pipeline_violation_here

File: .github/workflows/pluto-workflow-kustomize.yaml [hunk 2]
```diff
@@ -17,11 +17,11 @@ jobs:
...
       - name: Install all-in-one Kubernetes tools in a package.
-        uses: yokawasa/action-setup-kube-tools@v0.9.2
+        uses: yokawasa/action-setup-kube-tools@v0.9.3
```
[the remark this answers is not in the document]
Comment (martin-yang1234) [reply .github/workflows/pluto-workflow-kustomize.yaml:22]: #cycode_ignore_insecure_pipeline_violation_here

File: .github/workflows/pluto-workflow-shipit.yaml [hunk 3]
```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v3
       - name: Download Pluto
-        uses: FairwindsOps/pluto/github-action@v5.10.3
+        uses: FairwindsOps/pluto/github-action@v5.19.0
```
[the remark this answers is not in the document]
Comment (martin-yang1234) [reply .github/workflows/pluto-workflow-shipit.yaml:25]: #cycode_ignore_insecure_pipeline_violation_here

**Review by cttttt — APPROVED** [review]
👍🏽

**Pull request merged**